app/api/custom_forms.py

```diff
@@ -0,0 +1,102 @@
+from flask_rest_jsonapi import ResourceDetail, ResourceList, ResourceRelationship
+from marshmallow_jsonapi.flask import Schema, Relationship
+from marshmallow_jsonapi import fields
+
+from app.api.bootstrap import api
+from app.api.helpers.utilities import dasherize
+from app.api.helpers.permissions import jwt_required
+from app.models import db
+from app.models.custom_form import CustomForms
+from app.models.event import Event
+from app.api.helpers.db import safe_query
+
+
+class CustomFormSchema(Schema):
+    class Meta:
+        type_ = 'custom_form'
+        self_view = 'v1.custom_form_detail'
+        self_view_kwargs = {'id': '<id>'}
+        inflect = dasherize
+
+    id = fields.Integer(dump_only=True)
+    form = fields.Str(required=True)
+    type = fields.Str(default='text')
```

Comment:
@niranjan94 Updated. Please check.
(migrations and docs updated with required attributes)